ldrb/calculus.py

```diff
@@ -425,7 +296,10 @@ def compute_fiber_sheet_system(
     beta_epi_rv,
     beta_endo_sept,
     beta_epi_sept,
-    tol,
+    depth_tol=0.1,
+    tol_rv=0.01,
+    tol_lv=0.01,
+    tol_epi=0.05,
```

Comment:
As you point out, we have already experienced that these tolerances can be used to avoid issues with estimating the fibre orientations for some meshes, particularly in the region where the septum, left- and right ventricle walls intersect.

But I believe that it will be difficult to choose good default values for these options. I suspect that they may be very sensitive to the underlying mesh, and that it will not be obvious to users how they affect the generated fibres. (Moreover, a lot of trial-and-error may be required to understand how they should be chosen.)

So, if you do add them, I think you should probably also add a careful explanation about what they do. You may also want to consider disabling them by default.

Reply:
Good idea! I will add some more documentation 👍